plugins/wpe-headless/tests/acceptance/PostPreviewCest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+class PostPreviewCest
+{
+    /**
+     * Ensure the nodejs site url is set as the post preivew url.
+     */
+    public function i_can_view_the_post_preview_link(AcceptanceTester $I)
+    {
+        $post_title = 'Post Preview Post';
+        $post_name = 'post-preview-post';
+        $front_end_url = 'http://localhost:3000';
+
+        $I->haveWpeHeadlessSetting('frontend_uri', $front_end_url);
+        $I->haveWpeHeadlessSetting('secret_key', '3c042434-155d-4612-bce9-fe1de743dc6e');
```

Comment:
Should this be `00000000-0000-0000-0000-000000000001` to match the value in the .env.test?

Reply:
Yes..... how did that? hmmm. Nice catch.